projects/1.16/assets/redstone-gauges-and-switches/rsgauges/lang/zh_cn.json

```
@@ -1 +1,336 @@
+  "block.rsgauges.rustic_spring_reset_chain": "乡村式链式按钮",
+  "block.rsgauges.rustic_spring_reset_chain.help": "${rsgauges.switch.feature.typepulse.help}\n${rsgauges.switch.feature.outputconfig.help}\n${rsgauges.switch.feature.pulseextend.help}\n${rsgauges.switch.feature.pulsetime.help}\n${rsgauges.switch.feature.linktarget.help}\n${rsgauges.switch.feature.linksource.help}",
+  "block.rsgauges.rustic_two_hinge_lever": "乡村式双铰链拉杆",
+  "block.rsgauges.rustic_two_hinge_lever.help": "${rsgauges.switch.feature.typebistable.help}\n${rsgauges.switch.feature.outputconfig.help}\n${rsgauges.switch.feature.linktarget.help}\n${rsgauges.switch.feature.linksource.help}",
+  "block.rsgauges.sensitive_glass_block": "发光红石感应玻璃",
+  "block.rsgauges.sensitive_glass_block.help": "一种玻璃方块，被红石信号充能时会发光，与红石灯相似。",
+  "block.rsgauges.stained_sensitiveglass": "染色红石感应玻璃",
+  "block.rsgauges.stained_sensitiveglass.help": "一种玻璃方块，被红石信号充能时会从透明变成有色。微弱虚线边框代表该方块处于未充能状态。",
+  "block.rsgauges.valve_wheel_switch": "阀门开关",
+  "block.rsgauges.valve_wheel_switch.help": "${rsgauges.switch.feature.typebistable.help}\n${rsgauges.switch.feature.outputconfig.help}\n${rsgauges.switch.feature.linktarget.help}\n${rsgauges.switch.feature.linksource.help}",
+  "block.rsgauges.yellow_power_plant": "黄色充能植物",
+  "block.rsgauges.yellow_power_plant.help": "人造的红色花朵，和蒲公英长得很像，和接触板功能类似，当实体触碰到它时向底部和正面发出红石信号。\n${rsgauges.switch.feature.outputconfig.help}\n${rsgauges.switch.feature.contactmat.offdelay.help}\n${rsgauges.switch.feature.linksource.help}",
+  "item.rsgauges.switchlink_pearl": "开关连接珍珠",
+  "item.rsgauges.switchlink_pearl.help": "用末影珍珠左击开关获得。当使用（潜行右击）开关连接珍珠或插入另一个（源）开关时，将远程激活该（目标）开关。用开关连接珍珠右击目标来切换连接选项。你可以将多个珍珠插入开关。你可以听到珍珠在压力过大或找不到目标时的抱怨声。如果开关不适合作为连接源或目标，珍珠也会产生抱怨。不同的开关类型对触发连接的反应可能不同（只需稍微试验一下）。",
+  "rsgauges.gaugeconfig.options.comparator": "比较器输入",
+  "rsgauges.gaugeconfig.options.inverted": "反相",
+  "rsgauges.gaugeconfig.options.notinverted": "普通",
+  "rsgauges.subtitle.soundindicator1": "警报器：警报声",
+  "rsgauges.switch.feature.colortinting.help": "用任何染料左击开关进行染色。",
+  "rsgauges.switch.feature.contactmat.offdelay.help": "用红石左击来设置保持时间，范围在0.1秒（1个红石）到6.4秒（64个红石）之间。",
+  "rsgauges.switch.feature.contactmat.touch.help": "右击接触板的上/下按钮设置信号输出强度、实体过滤器，实体个数和灵敏度。",
+  "rsgauges.switch.feature.hoppermountable.help": "可以放置在漏斗上。",
+  "rsgauges.switch.feature.linksource.help": "用开关连接珍珠右击将其设置为连接源。",
+  "rsgauges.switch.feature.linktarget.help": "用末影珍珠右击来获得连接至此开关的开关连接珍珠。",
+  "rsgauges.switch.feature.outputconfig.help": "用红石火把右击设置输出（在开关支持的选项间循环）。",
+  "rsgauges.switch.feature.projectilesense.help": "可以被弹射物激活。",
+  "rsgauges.switch.feature.pulseextend.help": "在激活状态再次激活以延长脉冲时间。",
+  "rsgauges.switch.feature.pulsetime.help": "用红石左击来设置保持时间，范围在0.1秒（1个红石）到6.4秒（64个红石）之间。",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |  酌情考虑吧   |
```suggestion
  "rsgauges.switch.feature.pulsetime.help": "手持一定数量红石左击来设置脉冲时间，范围在0.1秒（1个）到6.4秒（64个）之间。",
```
原文：**Left-click with a stack of\n redstone to set a fixed pulse time\n between 0.1s (1) and 6.4s (64).**